airgun/widgets.py

```diff
@@ -2034,6 +2034,53 @@ def read(self):
         return self.progress
 
 
+class ProgressBarPF4(ProgressBar):
```

Comment:
I keep hitting this error: ```selenium.common.exceptions.NoSuchElementException: Message: Could not find an element Locator(by='xpath', locator='.//div[contains(@class,"progress-bar")]')```.  It seems sometimes the automation recognizes that it's on the progress bar modal page and other times it just doesn't recognize it and failing.  I'm not sure what the best approach for capturing that progressbar modal is.  Any help is appreciated.

@mshriver @omkarkhatavkar